ledger/participant-integration-api/src/main/scala/platform/store/CompletionFromTransaction.scala

```diff
@@ -60,4 +81,82 @@ private[platform] object CompletionFromTransaction {
       recordTime = Some(fromInstant(recordTime)),
       offset = Some(LedgerOffset.of(LedgerOffset.Value.Absolute(offset.toApiString))),
     )
+
+  private def toApiCompletion(
```

Comment:
It's mostly because of scalapb plus proto's "non-optional optionals"; I simplified it further in 2749c75 and I think a unit test at this level doesn't add much value because it's very thin logic and the test would basically duplicate it in order to check it. I think testing the `CompletionFromTransaction` unit at its interface (i.e., `acceptedCompletion` and `rejectedCompletion`) is in a similar situation as it just delegates to the private `toXXX` utilities.

Also, this logic should  already have full coverage by the DB-level integration test cases in `StorageBackendTestsCompletions`. WDYT?

Reply:
I'll merge this as soon as the build passes and we'll deal with further improvements in a separate PR, if need be.